conda_build/build.py

```diff
@@ -322,7 +322,7 @@ def build(m, get_src=True, verbose=True, post=None):
 
         # Display the name only
         # Version number could be missing due to dependency on source info.
-        print("BUILD START:", m.dist())
+        print("BUILD START:", m.name())
```

Comment:
Don't know -- that's what @ddale had done.  I believe it'll avoid giving bad information, however.  `m.name()` doesn't change, but the output of `m.dist()` does if git is involved.